geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/TestWithDataStore.scala

```diff
@@ -57,19 +57,21 @@ trait TestWithDataStore extends Specification {
   lazy val mockZookeepers = "myzoo"
   lazy val mockUser = "user"
   lazy val mockPassword = "password"
+
+  lazy val mockInstance = new MockInstance(mockInstanceId)
+
   // assign some default authorizations to this mock user
   lazy val connector: Connector = {
-    val mockInstance = new MockInstance(mockInstanceId)
     val mockConnector = mockInstance.getConnector(mockUser, new PasswordToken(mockPassword))
     mockConnector.securityOperations().changeUserAuthorizations(mockUser, MockUserAuthorizations)
     mockConnector
   }
 
   lazy val dsParams = Map(
-    "connector" -> connector,
-    "caching"   -> false,
+    AccumuloDataStoreParams.ConnectorParam.key -> connector,
```

Comment:
In another test, it seemed that the pattern was to import AccumuloDataStoreParams._ and use 'ConnectorParam.key' directly with less qualification.  

What should be preferred?

Reply:
i usually like to fully qualify them to make it clearer, but i'm not sure either way is really better